sgip_api.py

```diff
@@ -20,6 +20,7 @@
 # starttime and endtime are optional, if ommited will return the latest value
 starttime = '2020-03-01T00:00:00-0000'  # UTC offset of 0 (PDT is -7, PST -8)
 endtime = '2020-03-01T00:45:00-0000'
+moerversion = '1.0' #'2.0'

```

Comment:
this thing is just called 'version' in the api, so you should just call it 'version' or if you want to be specific (which is probably good), call it moer_version

I think 'moerversion' might be the param in /data or something?

Reply:
change to moer_version to be specific (and avoid confusion with forecast version)